test/unit/messaging/create-message.test.js

```diff
@@ -0,0 +1,29 @@
+const { BODY } = require('../../mocks/messaging/body')
+const { TYPE } = require('../../../app/constants/type')
+const { SOURCE } = require('../../../app/constants/source')
+
+const { createMessage } = require('../../../app/messaging/create-message')
+const { MESSAGE_ID } = require('../../mocks/messaging/message-id')
```

Comment:
Group the constants at the top of the file and group the mocks together - leaving a space between the constants and the mocks to separate them

Reply:
For things like this I blame co-pilot putting them where it wants haha